src/store/initialStates/tagsInitialState.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  addTags: {
+    loading: false,
+    response: '',
+    errors: {}
+  },
+  getTags: {
+    loading: false,
+    message: '',
+    errors: {}
+  }
```

Comment:
I think getting tags is already an article object. No need to fetch them again

Reply:
Yes yes already removed